JitsiConference.js

```diff
@@ -2166,7 +2170,7 @@ JitsiConference.prototype._startStopP2PSession = function(userLeftEvent) {
             }, peerCount: ${peerCount} => ${shouldBeInP2P}`);
 
     // Clear deferred "start P2P" task
-    if (!shouldBeInP2P && this.deferredStartP2P) {
+    if (!shouldBeInP2P && this.deferredStartP2PTask) {
```

Comment:
`_maybeClearDeferredStartP2P` already checks if `this.deferredStartP2PTask` is set, right? that's why it has the 'maybe'?  so we shouldn't need to check it here.